monitoring/osd-managing-metrics.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-metrics"]
+= Managing metrics
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: managing-metrics
+
+toc::[]
+
+This section provides an overview about how {product-title} metrics are collected, queried and visualized.
+
+// Understanding metrics
+include::modules/osd-monitoring-understanding-metrics.adoc[leveloffset=+1]
+
+// Setting up metrics collection for user-defined projects
+include::modules/osd-monitoring-setting-up-metrics-collection-for-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-deploying-a-sample-service.adoc[leveloffset=+2]
+include::modules/osd-monitoring-specifying-how-a-service-is-monitored.adoc[leveloffset=+2]
+
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+
+// Querying metrics
+include::modules/osd-monitoring-querying-metrics.adoc[leveloffset=+1]
+include::modules/osd-monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+include::modules/osd-monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the xref:../monitoring/osd-managing-metrics.adoc#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer] for details on accessing non-cluster metrics as a developer or a privileged user
+
+include::modules/osd-monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the xref:../monitoring/osd-managing-metrics.adoc#querying-metrics_managing-metrics[Querying metrics] section on using the PromQL interface
+* xref:../monitoring/osd-troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[Troubleshooting monitoring issues]
+
+[id="managing-metrics-next-steps"]
+.Next steps
+xref:../monitoring/osd-reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Reviewing monitoring dashboards]
```

Comment:
I skipped it originally because alerts are not supported. But I guess this is the RH way, so I added it. Maybe they'll add alerts in a future iteration.